set_test.go

```diff
@@ -137,3 +137,32 @@ func TestConcatSetType(t *testing.T) {
 		})
 	}
 }
+
+func TestConcatSetTypeElements(t *testing.T) {
+	t.Parallel()
+
+	tests := map[string][]SetDatatype{
```

Comment:
Can you make this a slice of struct please, so that the iteration order (and hence test run order) is deterministic?

Reply:
Good point.